### PR TITLE
added hw

### PR DESCRIPTION
**Pull request by rsln-ua**

### CODE REVIEW

File: src/classes/Park.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
+namespace Module2HW4;
+
+public class Park
+{
+    private int maxCountOfSections;
```
Comment (igorlopushko) [inline src/classes/Park.cs:5]: назови _maxCountOfSections

File: src/classes/Park.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
+namespace Module2HW4;
+
+public class Park
+{
+    private int maxCountOfSections; 
+    private int countOfSections = 0;
```
Comment (igorlopushko) [inline src/classes/Park.cs:6]: назови _countOfSections

File: src/classes/Section.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
+namespace Module2HW4;
+
+public class Section
+{
+    public Animal[] Animals = Array.Empty<Animal>();
+
+    public Section(params Animal[] animals)
```
Comment (igorlopushko) [inline src/classes/Section.cs:7]: для чего здесь params?

Comment (rsln-ua) [reply src/classes/Section.cs:7]: Prosto, shtob kogda sozdayu section, mog prosto peredaty argumentamy nuznoye kolichestvo animals, a pochemu imenno tak - dlya raznoobraziya))
https://github.com/rsln-ua/Module2HW4/blob/develop/src/classes/MockData.cs#L22

Eto nepravilyno nuzno peredelaty?

**Pull request merged**